WebContent/util/serverStatusCheckService.jsp

```diff
@@ -0,0 +1,6 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1"%>
+
+{
+	"HostName": "<%=request.getParameter("HostName")%>",
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg" width="20"/></sub>&nbsp; Cross-site Scripting (XSS)

Unsanitized input from `an HTTP parameter` flows into `print`, where it is used to render an HTML page returned to the user. This may result in a Cross-Site Scripting attack (XSS).

<h6>
Line 4 | <a href="https://cwe.mitre.org/data/definitions/79.html">CWE-79</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 792</a> | <a href="https://learn.snyk.io/lesson/xss/">Learn more about this vulnerability</h6>

<details>

<summary><b>Data flow: 3 steps</b></summary>

`Step 1 - 3` https://github.com/snykyzach/zach-goof/blob/f0d3e6e422831193b61ae2b06af494a79d37a464/WebContent/util/serverStatusCheckService.jsp#L4

</details>